src/main/java/org/lobid/resources/run/UpdateAliases.java

```diff
@@ -145,13 +145,12 @@ private static void deletedCountTest() throws IOException {
 				+ deletionsCount);
 		log(logMessage);
 		final int tolerance = (deletionsCount + 1) / 333;
-		logMessage =
-				("Going to compare if " + differenceDocsCountStagingVsProduction
-						+ " is less or equals " + deletionsCount
-						+ " allowing a 0.3% tolerance (i.e. " + tolerance + ")");
+		logMessage = ("Going to compare if " + differenceOfTotalHitsBetweenOldAndNew
+				+ " is less or equals " + deletionsCount
+				+ " allowing a 0.3% tolerance (i.e. " + tolerance + ")");
 		log(logMessage);
 		Assert.assertTrue(
-				deletionsCount - differenceDocsCountStagingVsProduction < tolerance);
+				differenceOfTotalHitsBetweenOldAndNew - deletionsCount < tolerance);
```

Comment:
I'm not sure I understand, is the the fix to swap the operands? Maybe I missed something, but should we not compare the *absolute* value to cover both a slightly larger and a slightly smaller new index? Something like `Math.abs(differenceOfTotalHitsBetweenOldAndNew - deletionsCount) < tolerance`?

Reply:
Hm , I think it's okay to have (many) *more* docs than expected. It's a different situation if you have *less*.

Reply:
Right, of course.